python/tests/unit/test_irauth.py

```diff
@@ -106,6 +107,40 @@ def test_irauth_grpcservice_version_v3():
     assert "mycoolauthservice.default.1" not in econf.ir.aconf.errors
 
 
+def test_cluster_fields():
+    yaml = """
+---
+apiVersion: getambassador.io/v3alpha1
+kind: AuthService
+metadata:
+  name:  mycoolauthservice
+  namespace: default
+spec:
+  auth_service: someservice
+  protocol_version: "v3"
+  proto: grpc
+  stats_name: authservice
```

Comment:
What happens if a user places a super long one? I know that cluster names are limited to I think 60 characters. Based on reading here: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto is says it follows the conventions of `upstream_cluster.{alt_stat_name}`....so if someone put a super long one would that blow up envoy or would envoy just trim itself?

Can you test that? Because if it is then I think we need to add some logic when the alt_stat_name is being set to trim it and log an error to the diagnostics.

Reply:
```
$ k exec -it emissary-cdf-69db9d9c74-tlv9v -- /bin/cat envoy/envoy.json | grep alt_stat
        "alt_stat_name": "fffffffffffffffffffffffffffffffffddddddddddddddddddddddddddddddddddddddddddsssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxffffffffffffffffoooooooooooooooooooooooooooooooooooooooopppppppppppppppppppppppppppppppppppppppqqqqqqqqqqqqqqqqqq",
```
No limitations there. Envoy picks it up with no issues. The limit of chars is in the cluster name, not the stats name.